inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -336,15 +332,20 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
     writeBits(header, os);
 
     for (auto &name : inputNames) {
-        writeBits(name.size(), os);
+        ulong test = sizeof(size_t);
+        writeBits(sizeof(size_t), os);
+        writeBits(name.size() + 1, os);
         writeNBytes(name.c_str(), name.size(), os);
```

Comment:
If we using c_str() then I would use strlen(name.c_str()) instead name.size()
as they can differ.
https://stackoverflow.com/questions/29302073/difference-between-strlenstr-c-str-and-str-length-for-stdstring